articles/healthcare-apis/includes/fhir-bulk-delete-operation.md

```diff
@@ -22,27 +22,28 @@ ms.custom: "include file"
 > [!NOTE]
 > Bulk delete is a an operation to be used with caution. Resources in FHIR service once deleted cannot be reverted.
 
-Bulk delete operation is currently in public preview. Review disclaimer for details.
-[!INCLUDE [public preview disclaimer](../includes/common-publicpreview-disclaimer.md)]
-
-
 ## Headers
 Bulk-Delete operation requires two header parameters  
 * Accept: application/fhir+json
 * Prefer: respond-async
 
+## Role
+To perform bulk delete operation, user or application needs to be assigned to FHIR Data Contributor role.
```

Comment:
They only need contributor for bulk hard delete (or purge history). FHIR Data Writer role can do bulk soft delete.